style.css

```diff
@@ -0,0 +1,41 @@
+@media (min-width: 320px) {
+	.wrap__title {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/CSS17K/pullRequest?prid=5813015)